src/lib.rs

```diff
@@ -32,6 +42,7 @@ mod test {
     use std::collections::HashMap;
 
     use rand::Rng;
+    use serde_json;
```

Comment:
`#[cfg(feature = "serde")]`? This causes a warning when running `cargo test` without `--features=serde`.

Reply:
Ah, I must have missed that when reshuffling imports. Fixing now.